code/__HELPERS/unsorted.dm

```diff
@@ -875,10 +875,11 @@ var/list/WALLITEMS_INVERSE = typecacheof(list(
 
 		var/list/cached_gases = air_contents.gases
 
-		for(var/id in cached_gases)
-			var/gas_concentration = cached_gases[id][MOLES]/total_moles
-			if(id in hardcoded_gases || gas_concentration > 0.001) //ensures the four primary gases are always shown.
-				user << "<span class='notice'>[cached_gases[id][GAS_META][META_GAS_NAME]]: [round(gas_concentration*100, 0.01)] %</span>"
+		for(var/id = 1 to GAS_LAST)
```

Comment:
This is pedantic as hell, but we do this format as ```for(var/id in X to Y)```, not ```for(var/i = X to Y)```, there's no (major) difference in performance between these two, but we prefer the former, so I'd prefer you swap to those, so people don't go "hurr is there a new format for loops?"